modules/stochastic_tools/test/tests/multiapps/commandline_control/master_multiple.i

```diff
@@ -20,10 +20,8 @@
 
 [MultiApps]
   [sub]
-    type = FullSolveMultiApp
-    positions = '0 0 0
-                 1 1 1
-                 2 2 2'
```

Comment:
Please don't change what tests do like this.

Reply:
I am not changing the gold file. The test is doing the same thing except the subapps is populated with positions before but now with the sampler. I will need to wait @aeslaughter to see if this change is ok. Now I am thinking that I can check the `use_positions` in base class to avoid this change actually. In that way, I also do not need to patch Mastodon.

Reply:
This is testing out a capability in MultiAppCommandLineControl so that you can use it to control a regular multi-app with a sampler. I agree please don't change this.

Reply:
Then I will be force to make `use_positions` to be a normal parameter otherwise the subapps will be always constructed earlier with `FullSolveMultiApp`. Does this sound ok to you?

Reply:
Or we can have an action in stochastic tools to set this private parameter to false at background for all multiapps.

Reply:
Talked with @aeslaughter this is a capability that really isn't needed. Using a sampler to control multiapps should use a sampler multiapp. Just make sure that the MultiApp `_use_positions` is false in MultiAppCommandLineControl.

Reply:
Thanks @aeslaughter and @zachmprince for this. It is always good to have a simpler design in the codes without hurting fundamental capabilities. I will add a check in the control as suggested.